rfcs/analytics/20230307-backend_service.md

```diff
@@ -84,3 +84,28 @@ provision all the following services:
 If we move to our own instance sometime in the future, we can
 [export data](https://plausible.io/docs/export-stats) and migrate that to our
 instance.
+
+## Pricing
+
+[Plausible pricing](https://plausible.io/#pricing) is based on number of events
+/ month. Each page view and each custom event recorded counts towards the total
```

Comment:
```suggestion
[Plausible pricing](https://plausible.io/#pricing) is based on number of page views
/ month. Each page view and each custom event recorded counts towards the total
```

Reply:
Ah, just read your next sentence and didn't get to delete my suggestion it time. Well, yes we'd quite quickly reach this then, especially considering we're already doing around 150k _page views_ a day.

Reply:
I'll reach out to the plausible folks and see if they have any recommendations. Since we have a single site and are a non-profit there might be some wiggle room to discuss alternative pricing.

There's also the 30 day trial, which we could spin up for development purposes at some point.

Reply:
<img width="982" alt="Screenshot 2023-03-08 at 4 48 57 PM" src="https://user-images.githubusercontent.com/16580576/223717361-fe8ab967-3fac-4b55-bd07-3bb05b11d266.png">